fmoe/megatron/utils.py

```diff
@@ -5,7 +5,7 @@ def add_fmoe_args(parser):
     group = parser.add_argument_group(title="fastmoe")
 
     group.add_argument("--fmoefy", action="store_true")
-    group.add_argument("--num-experts", type=int, default=None)
+    group.add_argument("--fmoe-num-experts", type=int, default=None)
```

Comment:
Maybe we can somehow keep `num_experts` for lower version of megatron to keep compatiblity?

Reply:
The reason on renaming num_experts to fmoe_num_experts is that num_experts is used in megatron v3.0 (which implements a switch transformer in a brute force way). 

I agree to keep compatibility, we can check megatron'v version in add_fmore_args to achieve that. I will update my pr.